Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,37 @@
 import React from 'react';
 import './App.css';
+import './components/DisplayComponents/CalculatorDisplay';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import ActionButton from './components/ButtonComponents/ActionButton';
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className='container'>
+      <div className='top'>
+        <CalculatorDisplay />
+      </div>
+      <div className='bottom'>
+          <ActionButton btnClassName="btn big" content='clear' />
```

Comment:
would there have been an easier way to do this instead of manually typing all of the components out and manually typing all of the content out? Think of the ES6 higher order function that would allow you to do this and how you could achieve only writing it one time.